### PR TITLE
[draft][4/5]Implement LRO unary rpc method's sample code

### PR DESCRIPTION
**Pull request by summer-ji-eng**

### CODE REVIEW

File: src/main/java/com/google/api/generator/gapic/composer/SampleCodeHelperComposer.java [hunk 1]
```diff
@@ -101,29 +102,73 @@ private static TryCatchStatement composeUnaryRpcMethodSampleCode(
...
+        .setTryBody(bodyStatements)
         .setIsSampleCode(true)
         .build();
   }
 
   private static TryCatchStatement composePagedUnaryRpcMethodSampleCode(
-      Method method, List<MethodArgument> arguments, TypeNode clientType) {
-    // TODO(summerji): compose sample code for unary paged rpc method.
-    // TODO(summerji): Add unit tests.
+      Method method,
```
Comment (miraleung) [inline src/main/java/com/google/api/generator/gapic/composer/SampleCodeHelperComposer.java:146]: Wasn't this part of the previous PR? Can we make the diffs from base PRs go away, so I can identify what I should be reviewing?

Will review the rest of the code afterwards.

**Review by miraleung — CHANGES_REQUESTED** [review]

**Comment by summer-ji-eng** [comment]
Rebased branch rpc_unary_paged_sc_33 and refactor client type to client variable expr.

File: src/main/java/com/google/api/generator/gapic/composer/SampleCodeHelperComposer.java [hunk 2]
```diff
@@ -97,16 +98,40 @@ private static TryCatchStatement composeUnaryRpcMethodSampleCode(
...
+                MethodInvocationExpr.builder()
+                    .setExprReferenceExpr(clientVarExpr)
+                    .setMethodName(getLroMethodName(method.name()))
+                    .setArguments(mapMethodArgumentsToVariableExprs(arguments))
+                    .build())
+            .setMethodName("get")
+            .setReturnType(method.outputType())
+            .build();
+    bodyExprs.add(
+        AssignmentExpr.builder()
```
Comment (summer-ji-eng) [inline src/main/java/com/google/api/generator/gapic/composer/SampleCodeHelperComposer.java:127]: TODO: fix consider method.outputType is VOID

File: test/integration/goldens/redis/CloudRedisClient.java [hunk 3]
```diff
@@ -758,7 +772,9 @@ public final Instance getInstance(GetInstanceRequest request) {
    *
    * <pre>{@code
    * try (CloudRedisClient cloudRedisClient = CloudRedisClient.create()) {
-   *   // Note: Not implemented yet, placeholder for lro Unary rpc method sample code.
+   *   String name = "name3373707";
+   *   InputConfig input_config = InputConfig.newBuilder().build();
```
Comment (miraleung) [inline test/integration/goldens/redis/CloudRedisClient.java:776]: Let's stick with the lowerCamelCase Java conventions.

File: test/integration/goldens/redis/CloudRedisClient.java [hunk 4]
```diff
@@ -758,7 +772,9 @@ public final Instance getInstance(GetInstanceRequest request) {
...
+   *   Operation response = cloudRedisClient.importInstanceAsync(name, input_config).get();
```
Comment (miraleung) [inline test/integration/goldens/redis/CloudRedisClient.java:777]: Could we please look for applicable examples where the proto would return empty, and handle those as well?

**Review by miraleung — CHANGES_REQUESTED** [review]
Will revisit this and the downstream PRs when the variable re-creation and comment accuracy comments from #512, #513 have been addressed.

**Comment by summer-ji-eng** [comment]
refactor in #587

**Pull request closed without merging**